tests/DobbleCardTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+use Marmelab\Dobble\Card;
+
+class DobbleCardTest extends \PHPUnit_Framework_TestCase
+{
+    public function validIterables()
+    {
+        return [
+            [[1, 2, 3]],
+            [['A', 'B', 'C']],
+            [['heart', 'square', 'diamond', 'club']],
+        ];
+    }
+
+    /**
+     * @dataProvider validIterables
+     */
+    public function testCardAcceptIterable($iterable)
```

Comment:
s/testCardAcceptIterable/testCardConstructorAcceptsIterable

same for the next method